examples/bri-3/src/bri/identity/bpiSubjects/capabilities/deleteBpiSubject/updateBpiSubjectCommand.handler.ts

```diff
@@ -0,0 +1,16 @@
+import { CommandHandler, ICommandHandler } from "@nestjs/cqrs";
+import { BpiSubjectAgent } from "../../agents/bpiSubjects.agent";
+import { BpiSubjectRepository } from "../../persistence/bpiSubjects.repository";
+import { DeleteBpiSubjectCommand } from "./deleteBpiSubject.command";
+
+@CommandHandler(DeleteBpiSubjectCommand)
```

Comment:
file name here is wrong (update...) and execute method is doing delete

Reply:
Fixed.